modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1225,6 +1213,28 @@ private PartitionReplicaListener createReplicaListener(
         );
     }
 
+    private CompletableFuture<Set<Assignment>> calculateAssignments(TablePartitionId tablePartitionId) {
+        // TODO: IGNITE-22661 Potentially unsafe to use the latest catalog version, as the tables might not already present
```

Comment:
Nice, that you've moved todo here. Initially I though that you've just removed it entirely.